### PR TITLE
BRD - UI: Table filters

### PR DESCRIPTION
**Pull request by deetz99**
issue: bcGov/entity#22569

- filter table results with header options
- set table to a fixed height 
- set sticky table header
- add comments to useAffiliations composable
- test filter options in useAffiliations composable
- set loading table prop
- set empty text in table

### CODE REVIEW

File: business-registry-dashboard/app/components/Table/Column/Header.vue [hunk 1]
```diff
@@ -11,9 +11,9 @@ defineProps<{
 defineEmits(['clear'])
 </script>
 <template>
-  <div class="flex flex-col gap-2 font-normal">
+  <div class="flex flex-col gap-2 border-b border-gray-200 font-normal">
```
Comment (deetz99) [inline business-registry-dashboard/app/components/Table/Column/Header.vue:14]: add border so theres a bottom border visible when scrolling the table, separating the thead and tbody

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-registry-dev--pr-17-1onmuwro.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-registry-dev--pr-17-1onmuwro.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**